app/components/reimbursement-item/template.hbs

```diff
@@ -0,0 +1,41 @@
+<li data-reimbursement-id={{@reimbursement.id}}
+    class="{{item-status @reimbursement}}"
+    {{on "click" this.toggleExpenseDetails}}>
```

Comment:
I'm guessing this is the offender why you had to disable the linter rule `no-invalid-interactive`.

Adding a `role="button"` to the list element should fix that.

```suggestion
<li data-reimbursement-id={{@reimbursement.id}}
    class="{{item-status @reimbursement}}"
    role="button"
    {{on "click" this.toggleExpenseDetails}}>
```